### PR TITLE
Update Issuing API preview

### PR DESCRIPTION
**Pull request by cko-devops**

### CODE REVIEW

File: nas_spec/components/schemas/issuing/card-management/cardholders/properties/IdentificationDocument.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        national_identity_card,
+        driving_license,
+        citizen_card,
+        residence_permit,
+        electoral_id,
+      ]
+    example: national_identity_card
+  front_document_id:
+    type: string
+    description: The ID of the front side of the document as represented within Checkout.com systems.
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/issuing/card-management/cardholders/properties/IdentificationDocument.yaml:22]:
```suggestion
    description: The ID on the front of the document, as represented within Checkout.com systems.
```

File: nas_spec/components/schemas/issuing/card-management/cards/get-card-credentials/get-encrypted-card-credentials-response.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+required:
+  - encrypted_block
+properties:
+  encrypted_block:
+    type: string
+    example: "ewogICJjYXJkX2lkcyI6IFsKICAgICJjcmRfZDNvemhmNDNwY3EyeGJsZG4yZzQ1cW5iNDQiLAogICAgImNyZF9xbTI2dDUycXozeWVqam8ydDRidHF2eHR6aSIKICBdCn0="
+    description:  |
+      Encrypted Card credentials
+
+      The encrypted_block is the encrypted representation of the Card Credentials [json](http://www.json.org/) document below.
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/issuing/card-management/cards/get-card-credentials/get-encrypted-card-credentials-response.yaml:11]:
```suggestion
      The `encrypted_block` is the encrypted representation of the Card Credentials [json](http://www.json.org/) document.
```
Suggest removing 'below' as directional language is inaccessible.

Do we need a json.org link here? Could we link instead to the Card Credentials document?

File: nas_spec/paths/issuing/cards@{cardId}@encrypted-credentials.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  - in: header
+    name: UserAgent
+    description: The combination of mobile OS and SDK versions. Provide along with
+      `Encryption-Key` if you want an RSA encrypted PIN returned.
+    example: ios-1.0.0
+    schema:
+      type: string
+  - in: header
+    name: Encryption-Key
+    description: 1024 bit RSA public key value (excluding ---BEGIN and ---END PUBLIC
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/paths/issuing/cards@{cardId}@encrypted-credentials.yaml:26]: Do ---BEGIN and ---END PUBLIC KEY--- need to formatted as code?

File: nas_spec/components/schemas/issuing/card-management/cardholders/properties/IdentificationDocument.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    example: file_6lbss42ezvoufcb2beo76rvwly
+  back_document_id:
+    type: string
+    description: The ID of the back side of the document as represented within Checkout.com systems. The back of the document is required for all document types apart from passports.
```
[the remark this answers is not in the document]
Comment (colm-prunty-cko) [reply nas_spec/components/schemas/issuing/card-management/cardholders/properties/IdentificationDocument.yaml:26]: Update [here](https://github.com/checkout/checkout-api-reference/pull/841), closing these extra ones

**Pull request closed without merging**